lib/rules/indentation/index.js

```diff
@@ -72,7 +72,7 @@ const rule = (primary, secondaryOptions = {}, context) => {
 
 			// Cut out any * and _ hacks from `before`
 			const before = (node.raws.before || '').replace(/[*_]$/, '');
-			const after = ('after' in node.raws && node.raws.after) || '';
+			const after = typeof node.raws.after === 'string' ? node.raws.after : '';
```

Comment:
[note] The `raws` objects like `AtRuleRaws` have inherited `Record<string, unknown>` with the new version of PostCSS, so type guards with the `in` operator are now insufficient.

- https://github.com/postcss/postcss/blob/8.3.11/lib/at-rule.d.ts#L3
- https://github.com/postcss/postcss/blob/8.4.5/lib/at-rule.d.ts#L3

```diff
-interface AtRuleRaws {
+interface AtRuleRaws extends Record<string, unknown> {
```